### PR TITLE
Image info and Customer Setup page introduced to the project environment.

### PR DESCRIPTION
**Pull request by AnshAggarwalMAC**
Please review the changes :
Screenshot 2024-02-29 at 5 20 00 AM
Screenshot 2024-02-29 at 6 39 59 AM

Screenshot 2024-02-29 at 7 23 22 AM
Screenshot 2024-02-29 at 7 23 56 AM

### CODE REVIEW

**Comment by AnshAggarwalMAC** [comment]
image
Screenshot 2024-02-29 at 11 10 10 AM

File: src/pages/CompanionSetup.tsx
Comment (anaikarsaad) [inline src/pages/CompanionSetup.tsx:1]: Wrong file

File: src/pages/ImageInfo.tsx
Comment (anaikarsaad) [inline src/pages/ImageInfo.tsx:1]: Wrong page

File: src/pages/YourImages.tsx
Comment (anaikarsaad) [inline src/pages/YourImages.tsx:1]: Wrong page

**Review by anaikarsaad — CHANGES_REQUESTED** [review]
Wrong pages done

**Comment by anaikarsaad** [comment]
Wrong screens

**Pull request closed without merging**